odata-protocol/11.7 Batch Requests.md

```diff
@@ -19,6 +19,17 @@ A batch request is represented using either the [multipart batch
 format](#MultipartBatchFormat) defined in this document or the JSON
 batch format defined in [OData-JSON](#ODataJSON).
 
+If the set of request headers of a batch request are valid the service
+MUST return a [`200 OK`](#ResponseCode200OK) HTTP response code to
+indicate that the batch request was accepted for processing, but the
+processing is yet to be completed. The individual requests within the
```

Comment:
That sounds as if the service is forced to send the status before sending the body.
```suggestion
indicate that the batch request was accepted for processing, even if the
processing is yet to be completed. The individual requests within the
```